pypodcastparser/Item.py

```diff
@@ -236,23 +232,27 @@ def set_itunes_episode(self, tag):
         """Parses the episode number and sets value"""
         try:
             self.itunes_episode = tag.string
+            if -32767 >= int(self.itunes_episode) or int(self.itunes_episode) >= 32767 or not self.itunes_episode:
+                self.itunes_episode = 0
         except AttributeError:
-            self.itunes_episode = None
+            self.itunes_episode = 0
 
     def set_itunes_season(self, tag):
         """Parses the episode season and sets value"""
         try:
             self.itunes_season = tag.string
+            if -32767 >= int(self.itunes_season) or int(self.itunes_season) >= 32767 or not self.itunes_season:
+                self.itunes_season = 0
         except AttributeError:
-            self.itunes_season = None
+            self.itunes_season = 0
 
     def set_itunes_episode_type(self, tag):
         """Parses the episode type and sets value"""
         try:
             self.itunes_episode_type = tag.string
             self.itunes_episode_type = self.itunes_episode_type.lower()
         except AttributeError:
-            self.itunes_episode_type = None
+            self.itunes_episode_type = 0
```

Comment:
We may need to be a little cautious with what we set it to on error, as setting it to 0 will cause it to be rejected by the DB.  The database only accepts `trailer`, `full`, or `bonus` per the DDL.

```    episode_type        text
        constraint episode_episode_type_check
            check (episode_type = ANY (ARRAY ['full'::text, 'trailer'::text, 'bonus'::text])),```

Reply:
Yep, that was a mistake on my part.